
Well, the usual fixes for big endian systems ...

  - Rene Rebe <rene@exactcode.de>

--- MPlayer-1.0pre6a/libao2/ao_jack.c	2004-12-07 03:24:15.000000000 +0100
+++ MPlayer-1.0pre6a-fixed/libao2/ao_jack.c	2005-03-08 15:22:38.000000000 +0100
@@ -153,7 +153,11 @@
 			m = 1;
 			break;
 		default:
+#ifndef __BIG_ENDIAN__
 			format = AFMT_S16_LE;
+#else
+			format = AFMT_S16_BE;
+#endif
 			bits_per_sample = 16;
 			m = 2;
 			break;
